README.md

```diff
@@ -6,7 +6,7 @@ structure of this project matches closely with existing Porter [Mixins](https://
 
 1. Create a new repository in GitHub [using this repository as a
    template](https://help.github.com/en/articles/creating-a-repository-from-a-template).
-1. We recommend using Go 1.12 without go modules.
+1. We recommend using Go 1.13 with go modules.
```

Comment:
It is required now, not recommended. So how about "Go 1.13 or higher is required. You can choose to clone into the GOPATH or not according to preference." 

We also need to instruct them to set `PKG = get.porter.sh/mixin/$(MIXIN)` in the makefile to match the new name of the package.

Reply:
Text around golang version updated.

The makefile should be good to go.  We have https://github.com/deislabs/porter-skeletor/blob/master/Makefile#L2 and the README has a step to find/replace `skeletor` -> `YOURNAME`...